### PR TITLE
Cropping

### PR DESCRIPTION
**Pull request by matthewrh**
- Added ability for users to crop images at Josh's preferred aspect ratio after upload
- Images are loaded and re-croppable at any time after submission
- Cropped images are now used everywhere in place of originals

### CODE REVIEW

**Comment by matthewrh** [comment]
See here: https://github.com/pennlabs/labs-api-server/pull/164

File: src/components/PostCard.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ class PostCard extends React.Component {
           <div className="column is-one-quarter">
             <div className="columns is-vcentered is-mobile" style={{padding: "0px 20px"}}>
               <div className="column is-flex" style={{alignItems: "center"}}>
-                <img src={props.imageUrl} style={{width: "142px", height: "79px"}}/>
+                <img src={props.imageUrl} style={{width: "142px", height: "79px"} /*props.imageUrlCropped ?*/}/>
```
Comment (joshdoman) [inline src/components/PostCard.js:28]: Can we remove this comment?

File: src/pages/Home.js [hunk 2]
```diff
@@ -20,9 +22,15 @@ class Home extends React.Component {
   }
 
   componentWillMount() {
-    var accountID = '7900fffd-0223-4381-a61d-9a16a24ca4b7'
+    var accountID = '7900fffd-0223-4381-a61d-9a16a24ca4b7' // 7900fffd-0223-4381-a61d-9a16a24ca4b7
```
Comment (joshdoman) [inline src/pages/Home.js:25]: Let's remove this comment.

**Review by joshdoman — APPROVED** [review]

**Comment by joshdoman** [comment]
Looks fine. Let's just remove some comments unless you think they're necessary. One thing I noticed is that the file is starting to get pretty big. In another pull request, let's abstract out some of the image parsing and other logic, perhaps into a model or other file.

**Pull request merged**